eng/common/scripts/job-matrix/job-matrix-functions.filter.tests.ps1

```diff
@@ -0,0 +1,84 @@
+Import-Module Pester
```

Comment:
nit: I wonder if it might be worth moving the tests files under a "tests" sub-directory to avoid any potential confusion with the real scripts/samples.